src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  person: { name, age, partnerName, sex, isMarried },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+    {isMarried ? (
+      <p className="Person__partner">
+        {`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
for better readability you can create separate variable in component like `partnerStatus` and use it in jsx
```suggestion
  const partnerStatus = sex === 'm'
     ? 'wife'
     : 'husband';
     ...
     `${partnerName} is my ${partnerStatus}`
```